hraven-core/src/main/java/com/twitter/hraven/rest/RestJSONResource.java

```diff
@@ -251,18 +323,39 @@ public Flow getJobFlowById(@PathParam("cluster") String cluster,
       builderIncludeConfigRegex.append(s);
     }
 
+    StringBuilder builderIncludeJobFields = new StringBuilder();
+    for(String s : includeJobFields) {
+      builderIncludeJobFields.append(s);
+    }
+
+    StringBuilder builderIncludeFlowFields = new StringBuilder();
```

Comment:
Same here (builderIncludeJobFields is identical to builderIncludeFlowFields). Any reason we need two variables and two operations to create them?